hoot-core/src/main/cpp/hoot/core/conflate/ConflateExecutor.cpp

```diff
@@ -487,27 +483,32 @@ void ConflateExecutor::_writeOutput(
       else
         statsFormat.setFormat(ChangesetStatsFormat::Text);
     }
-    _diffConflator.writeChangeset(
-      map, output, _diffConflateSeparateOutput, statsFormat, _osmApiDbUrl);
+    //  Merge in the tag changes if they aren't in supposed to be in a separate output
+    if (!_diffConflateSeparateOutput && _diffConflator.conflatingTags())
+    {
+      _diffConflator.addChangesToMap(map, _pTagChanges);
+      _currentTask++;
+    }
```

Comment:
This is the real significant change, the rest is formatting and minor Sonar issues.